src/app/contributors/page.tsx

```diff
@@ -0,0 +1,98 @@
+import Link from "next/link";
+
+async function getContributors() {
+    const res = await fetch("https://api.github.com/repos/TPOT-Hacks/website/contributors", {
```

Comment:
If we want to have this across the whole organization, we have to change this call.